src/java.base/share/classes/sun/launcher/resources/launcher_ja.properties

```diff
@@ -24,24 +24,27 @@
+<module>/<package>=<target-module>(,<target-module>)*\n                      モジュール宣言に関係なく、<module>を更新して<package>を<target-module>に\n                      エクスポートします。\n                      <target-module>をALL-UNNAMEDに設定すると、すべての名前のないモジュールに\n                      エクスポートできます。\n    --add-opens <module>/<package>=<target-module>(,<target-module>)*\n                      モジュール宣言に関係なく、<module>を更新して<package>を\n                      <target-module>に開きます。\n    --limit-modules <module name>[,<module name>...]\n                      参照可能なモジュールの領域を制限します\n    --patch-module <module>=<file>({0}<file>)*\n                      JARファイルまたはディレクトリのクラスおよびリソースで\n                      モジュールをオーバーライドまたは拡張します。\n    --source <version>\n                      ソースファイル・モードでソースのバージョンを設定します。\n    --finalization=<value>\n                      JVMがオブジェクトのファイナライズを実行するかどうかを制御します\n                      <value>は"enabled"または"disabled"のいずれかです。\n                      ファイナライズはデフォルトで有効になっています。\n    --sun-misc-unsafe-memory-access=<value>\n                      サポートされていないAPI sun.misc.Unsafeの使用を許可または拒否します\n                      <value>は"allow"、"warn"、"debug"または"deny"のいずれかです。\n                      デフォルト値は"warn"です。\n\nこの追加オプションは予告なしに変更されることがあります。\n
 
 # Translators please note do not translate the options themselves
 java.launcher.X.macosx.usage=\n次のオプションはmacOS固有です:\n    -XstartOnFirstThread\n                      main()メソッドを最初(AppKit)のスレッドで実行する\n    -Xdock:name=<application name>\n                      Dockに表示されるデフォルト・アプリケーション名をオーバーライドする\n    -Xdock:icon=<path to icon file>\n                      Dockに表示されるデフォルト・アイコンをオーバーライドする\n\n
 
+# Translators please note do not translate the options themselves
+java.launcher.opt.concise.header  =   使用方法: java [java options...] <application> [application arguments...]\n\n<application>は次のいずれかです:\n  <mainclass>                コンパイルされたメイン・クラスのメイン・メソッドを実行します\n  -jar <jarfile>.jar         JARアーカイブのメイン・クラスを実行します\n  -m <module>[/<mainclass>]  モジュールのメイン・クラスを実行します\n  <sourcefile>.java          ソースファイル・プログラムをコンパイルおよび実行します\n\n主要なjavaオプションには次が含まれます:\n  --class-path <class path>\n    <class path>は、クラス・ファイルを検索するためのディレクトリおよびJARアーカイブのリストであり、"{0}"で区切られます\n  --module-path <module path>\n    <module path>は、モジュールを検索するためのディレクトリおよびJARアーカイブのリストであり、"{0}"で区切られます\n  -version\n    製品バージョンをエラー・ストリームに出力し終了します\n\n使用方法についての追加のヘルプの場合:           java --help\n対話型のJava環境の場合:    jshell
```

Comment:
Similarly to the German version, the currently of `java` and `jshell` are not aligned:
```
使用方法: java [java options...] <application> [application arguments...]

<application>は次のいずれかです:
  <mainclass>                コンパイルされたメイン・クラスのメイン・メソッドを実行します
  -jar <jarfile>.jar         JARアーカイブのメイン・クラスを実行します
  -m <module>[/<mainclass>]  モジュールのメイン・クラスを実行します
  <sourcefile>.java          ソースファイル・プログラムをコンパイルおよび実行します

主要なjavaオプションには次が含まれます:
  --class-path <class path>
    <class path>は、クラス・ファイルを検索するためのディレクトリおよびJARアーカイブのリストであり、":"で区切られます
  --module-path <module path>
    <module path>は、モジュールを検索するためのディレクトリおよびJARアーカイブのリストであり、":"で区切られます
  -version
    製品バージョンをエラー・ストリームに出力し終了します

使用方法についての追加のヘルプの場合:           java --help
対話型のJava環境の場合:    jshell
```

It would be nicer if they did (assuming it is doable):
```
使用方法: java [java options...] <application> [application arguments...]

<application>は次のいずれかです:
  <mainclass>                コンパイルされたメイン・クラスのメイン・メソッドを実行します
  -jar <jarfile>.jar         JARアーカイブのメイン・クラスを実行します
  -m <module>[/<mainclass>]  モジュールのメイン・クラスを実行します
  <sourcefile>.java          ソースファイル・プログラムをコンパイルおよび実行します

主要なjavaオプションには次が含まれます:
  --class-path <class path>
    <class path>は、クラス・ファイルを検索するためのディレクトリおよびJARアーカイブのリストであり、":"で区切られます
  --module-path <module path>
    <module path>は、モジュールを検索するためのディレクトリおよびJARアーカイブのリストであり、":"で区切られます
  -version
    製品バージョンをエラー・ストリームに出力し終了します

使用方法についての追加のヘルプの場合:  java --help
対話型のJava環境の場合:             jshell
```